python/source/org/libreoffice/ide/eclipse/python/PythonBuilder.java

```diff
@@ -112,22 +93,6 @@ public IFile createLibrary(IUnoidlProject pUnoProject) throws Exception {
     public void generateFromTypes(ISdk pSdk, IOOo pOoo, IProject pPrj, File pTypesFile,
         File pBuildFolder, String pRootModule, IProgressMonitor pMonitor) {
 
-        if (pTypesFile.exists()) {
-
-            if (null != pSdk && null != pOoo) {
-
-                String[] paths = pOoo.getTypesPath();
-                String oooTypesArgs = ""; //$NON-NLS-1$
-                for (String path : paths) {
-                    IPath ooTypesPath = new Path(path);
-                    oooTypesArgs += " -X\"" + ooTypesPath.toOSString() + "\""; //$NON-NLS-1$ //$NON-NLS-2$
-                }
-
-                // TODO What if the user creates other root modules ?
-                String firstModule = pRootModule.split("::")[0]; //$NON-NLS-1$
-
-            }
-        }
```

Comment:
The code that was not required in the creation/Running/Export of Python Project have been removed.